StaticAnalyzer/views/android/converter.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf_8 -*-
+"""Module holding the functions for converting."""
+
+import platform
+import subprocess
+import os
+
+from django.conf import settings
+
+from StaticAnalyzer.views.android.win_fixes import (
+    win_fix_python3,
+    win_fix_java
+)
+
+from MobSF.utils import (
+    PrintException,
+    isFileExists,
+    isDirExists
+)
+
+def dex_2_jar(app_path, app_dir, tools_dir):
+    """Run dex2jar."""
+    try:
+        print "[INFO] DEX -> JAR"
+        args = []
+        working_dir = False
+        if settings.JAR_CONVERTER == "d2j":
+            print "[INFO] Using JAR converter - dex2jar"
+            if platform.system() == "Windows":
+                win_fix_java(tools_dir)
+                d2j = os.path.join(tools_dir, 'd2j2/d2j-dex2jar.bat')
+            else:
+                inv = os.path.join(tools_dir, 'd2j2/d2j_invoke.sh')
+                d2j = os.path.join(tools_dir, 'd2j2/d2j-dex2jar.sh')
+                subprocess.call(["chmod", "777", d2j])
+                subprocess.call(["chmod", "777", inv])
+            if len(settings.DEX2JAR_BINARY) > 0 and isFileExists(settings.DEX2JAR_BINARY):
+                d2j = settings.DEX2JAR_BINARY
+            args = [
+                d2j,
+                app_dir + 'classes.dex',
+                '-f',
+                '-o',
+                app_dir + 'classes.jar'
+            ]
+        elif settings.JAR_CONVERTER == "enjarify":
+            print "[INFO] Using JAR converter - Google enjarify"
+            if len(settings.ENJARIFY_DIRECTORY) > 0 and isDirExists(settings.ENJARIFY_DIRECTORY):
+                working_dir = settings.ENJARIFY_DIRECTORY
+            else:
+                working_dir = os.path.join(tools_dir, 'enjarify/')  # TODO(Need to understand this)
+            if platform.system() == "Windows":
+                win_fix_python3(tools_dir)
+                enjarify = os.path.join(working_dir, 'enjarify.bat')
+                args = [enjarify, app_path, "-f", "-o", app_dir + 'classes.jar']
+            else:
+                working_dir = True
```

Comment:
I think this is a bug introduce by me. I will fix it after merging. 
`working_dir` is assigned both to string and boolean